petab/lint.py

```diff
@@ -404,6 +405,8 @@ def assert_model_parameters_in_condition_or_parameter_table(
             continue
         if parameter_id.startswith('noiseParameter'):
             continue
+        if parameter.getConstant():
+            continue
```

Comment:
I hope that's always set correctly then in the sbml models (don't remember the constant attribute being documented somewhere).

Reply:
constant parameters means that they cannot be assigned by AssignmentRules. It's a little bit of an abuse of notations.

Reply:
It is... Not sure how to handle it better. Alternatives: 1) removing that check  2) requiring ALL model parameters to be specific in the parameter table, which becomes even more redundant with the SBML file. So probably (1)